build/config/rmi_pacta_2023q4_pa2024ch.json

```diff
@@ -1,12 +1,14 @@
 {
-  "data_share_path": "2023Q4_20240424T120055Z",
+  "data_share_path": "2023Q4_20240709T111731Z",
   "pacta_data_quarter": "2023Q4",
   "project_code": "PA2024CH",
   "templates_ref": "",
-  "peer_results": "https://pactadatadev.blob.core.windows.net/testing-files/peer-results/PA2024CH_peer-results-test",
+  "peer_results": "https://pactadatadev.blob.core.windows.net/project-files/pa2024ch/peer_files",
```

Comment:
I know it's mentioned in the (very long) thread of this PR, but it's not super clear to me why/if the peer files should change here.

Reply:
Would you like me to document that in the PR body?